__tests__/app.test.js

```diff
@@ -31,9 +33,560 @@ describe("NCNews-Server Unit Tests", () => {
+                        const articlesArray = response.body.articles;   
+                        expect(articlesArray).toHaveLength(12);
+        
+                        articlesArray.forEach((e, index) => {
+                            if (index < articlesArray.length - 1) {
+                            expect(Date.parse(articlesArray[index][`created_at`]) > Date.parse(articlesArray[index + 1][`created_at`])).toBe(true);
+                        }});
+                    });
+                });
+
+                test(`[ 404 ] Responds with an error when passed invalid parameters, test 1`, () => {
+
+                    return request(app).get(`/api/articles?order=sdfdefds`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+
+                test(`[ 404 ] Responds with an error when passed invalid parameters, test 2`, () => {
+
+                    return request(app).get(`/api/articles?order=1337`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+        
+                test(`[ 404 ] Responds with an error when passed an SQL injection test 1`, () => {
+                    
+                    return request(app).get(`/api/articles?order=1&#59;&nbsp;DROP&nbsp;TABLE&nbsp;articles`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+        
+                test(`[ 404 ] Responds with an error when passed an SQL injection test 2`, () => {
+                    return request(app).get(`/api/articles?order=asc; DROP TABLE articles`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+
+                
+               
+            });
+
+            describe(`GET /api/articles/?sort_by=`, () => {
+
+                test(`[ 200 ] When passed "order=desc" as a URL parameter, and "sort_by=author"...`, () => {
+
+                    return request(app).get(`/api/articles?order=desc&sort_by=author`).expect(200).then((response) => {
+                        const articlesArray = response.body.articles;   
+                        expect(articlesArray).toHaveLength(12);
+                        expect(articlesArray[0]["author"]).toBe("rogersop");
+                        expect(articlesArray[3]["author"]).toBe("icellusedkars");
+                        expect(articlesArray[articlesArray.length - 1]["author"]).toBe("butter_bridge");
+                        
+                      });
+                });
+
+                test(`[ 200 ] When passed "order=asc" as a URL parameter, and "sort_by=author"...`, () => {
+
```

Comment:
you've tested for invalid orders but not sort_bys, what happens if I try to sort_by=bananas ?